pages/overview/installation.mdx

```diff
@@ -48,21 +51,26 @@ AUTH_SECRET="This is an example"
 
 `AUTH_SECRET` is a random token used by the library to encrypt tokens and email verification hashes, and it's mandatory to keep things secure (See [Deployment](/overview/deployment) to learn more). On Unix based systems you can use this command:
 
+  </Code.Next>
+  <Code.Svelte />
+  <Code.Solid />
+  <Code.Express />
+</Code>
+
```

Comment:
No need to call `<Code.Svelte />`, etc... here. 

This `<Code />` component is wired magically so that it will always render "empty" tabs for **Next.js**, **Svelte**, **Solid** and **Express** 🪄 , ( the frameworks we support ) so most of the time we can just do:
```suggestion
<Code>
  <Code.Next>
    ...
   </Code.Next>
</Code>
```
And if the supported frameworks are committed it'll magically render a tab with the empty state for them. I need to document it better 😪 

The exemption is `Next.js (pages)` where, if you don't supply a code example, it'll not render an empty tab (as Next.js pages we want to document just in certain scenarios, where it makes sense).

Reply:
Ahh right, perfect - I've removed the empty ones then :+1: